ext/storage_sources/gcp_store/gcp_log_system.cpp

```diff
@@ -0,0 +1,67 @@
+/*-
+ * Public Domain 2014-present MongoDB, Inc.
+ * Public Domain 2008-2014 WiredTiger, Inc.
+ *
+ * This is free and unencumbered software released into the public domain.
+ *
+ * Anyone is free to copy, modify, publish, use, compile, sell, or
+ * distribute this software, either in source code form or as a compiled
+ * binary, for any purpose, commercial or non-commercial, and by any
+ * means.
+ *
+ * In jurisdictions that recognize copyright laws, the author or authors
+ * of this software dedicate any and all copyright interest in the
+ * software to the public domain. We make this dedication for the benefit
+ * of the public at large and to the detriment of our heirs and
+ * successors. We intend this dedication to be an overt act of
+ * relinquishment in perpetuity of all present and future rights to this
+ * software under copyright law.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+ * OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ */
+#include "gcp_log_system.h"
+#include <cstdarg>
+
+// Constructor for gcp_log_system that calls to set the WiredTiger verbosity level.
```

Comment:
I'm not a huge fan of the comments - in general they should explain why, not how (since the "how" is clear from the source). There's not much "why" to explain in this code, so I'd prefer to remove the ones that just repeat what the code says. I'm happy to discuss this offline if you disagree.

Reply:
Yeah I have removed all of it now, I agree with what you said here. Let me know your thoughts! [6fdbcfd](https://github.com/wiredtiger/wiredtiger/pull/8868/commits/6fdbcfdd84795aba19ff2558578738a6d78dde15)

Reply:
Thanks, that's perfect now.